.nvmrc

```diff
@@ -1 +1 @@
-22.12.0
+22.13.0
```

Comment:
Didn't realize these repos are on v22: should we add them to the github action environment? https://github.com/turo/packer-github-action-general/pull/150

Reply:
We should, although the CI for these is in the default runners, and in a consumer repo, the used node version is the one that that repo sets